### ISSUE TITLE
Solve a problem of loading a SDF file with a kinematic loop constraint

### ISSUE THREAD

**Issue by XiaoWZENG** [stack-of-tasks/pinocchio#2041]
Hey, can I use the pinocchio3-preview version to load a SDF file with a kinematic loop constraint?  If it does work, is there any demo. Thanks a lot.

**Comment by jcarpent** [comment]
Yes, you can. We have been able to use it for some robots already.

**Comment by XiaoWZENG** [comment]
Wow, that's great. Could you tell me where should I find these examples? The pinocchio3-preview library does not contain such a exapmle.

**Comment by TJBetter** [comment]
Hello @jcarpent any update on this topic?
